metaseq/data/document_to_sequence.py

```diff
@@ -0,0 +1,352 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import math
+from typing import Optional
+
+import numpy as np
+import torch
+
+from metaseq.data import data_utils
+import time
+from metaseq.data.atomic_array import AtomicArray
+
+
+# gotta be a better way to get this...
+def f():
```

Comment:
Wut

Reply:
I was using this to get to Pythons 'cell' type which it uses for closed over variables, but I think this is extra confusing. I replaced the Cell's with single-element lists (at first  list[0] holds the `create` function to load token data, once loaded, list[0] holds the tokenized tensor).